scalikejdbc-mapper-generator-core/src/main/scala/scalikejdbc/mapper/CodeGenerator.scala

```diff
@@ -376,6 +376,34 @@ class CodeGenerator(table: Table, specifiedClassName: Option[String] = None)(imp
         createColumns.map { c => 3.indent + c.nameInScala + " = " + c.nameInScala }.mkString(comma + eol) + ")" + eol +
         1.indent + "}" + eol
     }
+    /**
+     * {{{
+     * def batch(list: List[Member])(implicit session: DBSession = autoSession) = {
+     *   val params: Seq[Seq[(Symbol, Any)]] = list.map(elem =>
+     *   Seq(
+     *     'id -> entity.id,
+     *     'name -> entity.name,
+     *     'birthday -> entity.birthday)
+     *   SQL("""insert into member values (
+     *     {id},
+     *     {name},
+     *     {birthday}
+     *   )""").batchByName(params: _*).apply()
+     * }
+     * }}}
+     */
+    val batchMethod = {
+      // def batch=(
+      1.indent + s"def batch(list: List[" + className + "])(implicit session: DBSession = autoSession) {" + eol +
```

Comment:
For historical reason, findAll returns List value but I think accepting Seq[Entity] here is reasonable enough.